S2VX.Game/Editor/EditorScreen.cs

```diff
@@ -325,6 +332,8 @@ public class EditorScreen : Screen {
             Story.Save(@"../../../story.json");
         }
 
+        private void ProjectQuit() => this.Exit();
```

Comment:
Pressing escape while the song is playing causes the song to continue playing

Reply:
Added Track to EditorScreen's InternalChildren so it is removed properly when EditorScreen exits